praktikumsplaner-frontend/frontend/src/api/FetchUtils.ts

```diff
@@ -95,11 +99,42 @@ export default class FetchUtils {
                 });
             } else if (response.type === "opaqueredirect") {
                 location.reload();
+            } else if (response.status >= 400 && response.status <= 499) {
```

Comment:
with a function like `isStatusOfInput(response.status)` u can omit the comment above?